### ISSUE TITLE
Allow users to express fused matmul/bias/relu

### ISSUE THREAD

**Issue by swolchok** [pytorch/pytorch#77171]
### 🚀 The feature, motivation and pitch

We should let users have the benefit of `_addmm_activation` (fused matmul/bias/relu; also matmul/bias/gelu if/when that is fixed in cublasLt) from #74490 in their models. Some ideas on how we might do this:

- add a LinearReLU module that calls addmm + relu in training and just _addmm_activation in inference
- add a backward & documentation for _addmm_activation and make it public
- both?
- something else?

CC @ngimel 

### Alternatives

- Do nothing and rely on nvfuser; doesn't help eager mode
- Do nothing; doesn't get maximum performance

**Comment by ngimel** [comment]
cc @cpuhrsch

**Comment by ngimel** [comment]
@vadimkantorov currently we have only fused mm - bias - relu, cublasLt probably allows other variants but we haven't implemented them yet (and looking at how mm enablement went, it won't be an easy process because cublasLt is very buggy).

**Comment by cpuhrsch** [comment]
Yes, I agree with this. See POC under https://github.com/cpuhrsch/torchfused , which is based on [xformers](https://github.com/facebookresearch/xformers/tree/main/xformers).
